packages/web-twig/src/Resources/components/Grid/Grid.twig

```diff
@@ -21,7 +23,50 @@
     {%- set _colsClasses = [_spiritClassPrefix ~ 'Grid--cols-' ~ _cols] -%}
 {%- endif -%}
 
-{%- set _classNames = [ _rootClassName, _styleProps.className ] | merge(_colsClasses) -%}
+{%- set _alignmentXClasses = [] -%}
+{%- if _alignmentX is iterable and _alignmentX is not null -%}
+    {%- set hasMobile = false -%}
+    {%- for breakpoint, breakpointValue in _alignmentX -%}
+        {%- if breakpoint == 'mobile' -%}
+            {%- set hasMobile = true -%}
+        {%- endif -%}
+    {%- endfor -%}
+    {%- if not hasMobile -%}
+        {%- set _alignmentX = _alignmentX | merge({ 'mobile': 'stretch' }) -%}
```

Comment:
The HTML output is nicer when you swap the two:

```suggestion
        {%- set _alignmentX = { 'mobile': 'stretch' } | merge(_alignmentX) -%}
```

Now the class names are listed in the same order everywhere 🙂.